hieradata_aws/staging.yaml

```diff
@@ -165,8 +165,8 @@ govuk::apps::email_alert_api::email_address_override: 'success@simulator.amazons
 govuk::apps::email_alert_api::email_address_override_whitelist_only: true
 govuk::apps::email_alert_api::delivery_request_threshold: "3000"
 govuk::apps::email_alert_frontend::subscription_management_enabled: true
-govuk::apps::finder_frontend::nagios_memory_critical: 4500
-govuk::apps::finder_frontend::nagios_memory_warning: 3500
+govuk::apps::finder_frontend::nagios_memory_critical: 6000
+govuk::apps::finder_frontend::nagios_memory_warning: 5000
 govuk::apps::finder_frontend::unicorn_worker_processes: "12"
```

Comment:
From the commit message, it sounds like you wanted to increase this number also?

Reply:
sorry, human error. have amended the commit message and PR title